### ISSUE TITLE
Custom tickets: chat notifications

### ISSUE THREAD

**Issue by carmenfan** [3drepo/3drepo.io#3577]
### Description
We need chat notifications for:
   - When a new ticket has been added to a federation/container
   - When an update has been applied with an existing ticket

#### New ticket (Container)
- Event name: `containerNewTicket`
- scope: Container/Federation event
- data: 
````js
{
  teamspace: "teamspace",
  project: "projectId",
  model: "federationId",
  data: {
    //All the data we need for adding an entry to the list https://github.com/3drepo/3drepo.io/issues/3575
  }
}
````

#### New ticket (Federation)
- Event name: `federationNewTicket`
- scope:  Container/Federationevent
- data: 
````js
{
  teamspace: "teamspace",
  project: "projectId",
  model: "federationId",
  data: {
    //All the data we need for adding an entry to the list https://github.com/3drepo/3drepo.io/issues/3575
  }
}
````

#### Update ticket (Container)
- Event name: `containerUpdateTicket`
- scope:  Container/Federationevent
- data: 
````js
{
  teamspace: "teamspace",
  project: "projectId",
  model: "containerId",
  data: {
    _id: "ticket id",
    // plus any fields that has been updated, e.g.:
   title: "New title",
   properties: {
     Priority: "High"
   }
  }
}
````

#### Update ticket (Federation)
- Event name: `federationUpdateTicket`
- scope:  Container/Federationevent
- data: 
````js
{
  teamspace: "teamspace",
  project: "projectId",
  model: "federationId",
  data: {
    _id: "ticket id",
    // plus any fields that has been updated, e.g.:
   title: "New title",
   properties: {
     Priority: "High"
   }
  }
}
````

#### NOTES:
- This issue should branch off #3564 
- Data should be serialised (you will probably need changes from #3545, refactor the serialise function out of middleware potentially..)
- Ensure read only fields are also passed over if they are added/updated (e.g. Updated At, level of risk)

### Goals
- [ ] Add the aforementioned chat notifications
- [ ] Update chat server documentation

### Tasks
to be filled in by the developer

### Related Resources
This issue depends on #3564
